schedule/yam/agama_auto_multipath.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
instead of auto, use always `unattended` for file name.

Reply:
changed.